elyra/templates/kubeflow/v2/python_dsl_template.jinja2

```diff
@@ -18,40 +18,7 @@ from kfp.dsl import PipelineTask
 from kfp.kubernetes import common
 from kfp.kubernetes import kubernetes_executor_config_pb2 as pb
 
-try:
-    from typing import Literal
-except ImportError:
-    from typing_extensions import Literal
-
-
-def add_pod_label(
-    task: PipelineTask,
-    label_key: str,
-    label_value: str,
-) -> PipelineTask:
-
-    msg = common.get_existing_kubernetes_config_as_message(task)
-    msg.pod_metadata.labels.update({label_key: label_value})
-    task.platform_config['kubernetes'] = json_format.MessageToDict(msg)
-
-    return task
-
-
-def add_pod_annotation(
-    task: PipelineTask,
-    annotation_key: str,
-    annotation_value: str,
-) -> PipelineTask:
-
-    msg = common.get_existing_kubernetes_config_as_message(task)
-    msg.pod_metadata.annotations.update({annotation_key: annotation_value})
-    task.platform_config['kubernetes'] = json_format.MessageToDict(msg)
-
-    return task
-
-# ------------------------------------------------------------------
-# end of missing functions
-# ------------------------------------------------------------------
```

Comment:
I guess this removal isn't part of the rest of the PR? If so, should we have it in a separate commit/PR for the cleaner approach?

Reply:
i believe it can stay here, but perhaps we can change the commit message, to include all.